### DISCUSSION TITLE
only 'ncc_c, ncc_e' cookies visible?

### DISCUSSION THREAD

**Discussion by Asko-Dev** [dargmuesli/nuxt-cookie-control#167] [Q&A] [answered]
Hi, I have to be doing something wrong or just not understanding the concept. I did everything according to the tutorial - my nuxt-config below

At least I hope so ... but whenever I click "accept" the only cookies shown in Application Cookies upon inspect are ncc_c and ncc_e  whose values are a combination of the id of necessary and optional cookies ... is this correct? I thought I would see the ids in the targetCookieIds spelled out in the Cookies list. 

<img width="448" alt="Screenshot 2023-11-16 at 16 01 32" src="https://github.com/dargmuesli/nuxt-cookie-control/assets/57365135/8e5d735d-5972-44cd-a747-95513750ded2">

```cookieControl: {
        locales: ["cs"],
        closeModalOnClickOutside: true,
        cookies: {
            necessary: [
                {
                    name: "Website & API",
                    description: { cs: "Nezbytne nutne cookies pro spravne fungovani stranky" },
                    targetCookieIds: ["ncc_c", "ncc_e"],
                },
            ],
            optional: [
                {
                    id: "google-analytics",
                    name: { cs: "Advanced Cookies" },
                    description: {
                        cs: "Used to help website owners understand how visitors interact with the website",
                    },
                    targetCookieIds: ["_ga", "_gat", "ga<container-id>", "_gid"],
                },
            ],
        },
        colors: {
            barBackground: "#ffffff",
            barButtonBackground: "rgb(var(--v-theme-primary))",
            barButtonHoverBackground: "rgb(var(--v-theme-primShade4))",
            barButtonColor: "#000",
            barTextColor: "#000",
            controlButtonHoverBackground: "rgb(var(--v-theme-primShade4))",
            controlButtonIconColor: "rgb(var(--v-theme-primary))",
            modalButtonBackground: "rgb(var(--v-theme-primary))",
            modalButtonHoverBackground: "rgb(var(--v-theme-primShade4))",
            modalButtonColor: "#000",
            checkboxActiveBackground: "rgb(var(--v-theme-secondary))",
        },
    }`

Pls help.

**Comment by dargmuesli** [comment, marked as answer]
Those cookies are just saving the user's preference on which cookies to consent to. It's up to you to implement the cookies that were given consent to be stored in a way that respects the user's preferences. For that you can make use of this library's functions.

**Comment by Asko-Dev** [comment]
I implemented the nuxt gtm plugin and even though it seems like it's working (sends debugs messages correctly), once I enable it based on the cookies it just doesn't add the google analytics etc. cookies to the list :/ the container ID is correct for the domain I am using.... but nothing appears

**Reply by dargmuesli** [reply]
Please provide a [minimal reproduction](https://nuxt.new/) for me to understand what e.g. "enable it based on the cookies" means.
